.github/pull_request_template.md

```diff
@@ -12,9 +12,9 @@ Briefly list the changes made to the code:
 
 ### ✔️ Verification
 
-What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves. 
+What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.
```

Comment:
Same comment as above, let's not have minimal changes here that don't relate to scope of the ticket